test/DurableTaskEndToEndTests.cs

```diff
@@ -443,6 +443,43 @@ public DurableTaskEndToEndTests(ITestOutputHelper output)
         }
 
         /// <summary>
+        /// End-to-end test which validates the handling of unhandled exceptions generated from orchestrator code.
+        /// </summary>
+        [Fact]
+        public async Task Orchestration_Activity()
+        {
+            string[] orchestratorFunctionNames =
+            {
+                nameof(TestOrchestrations.OrchestratorGreeting),
+                nameof(TestOrchestrations.SayHelloWithActivity)
+            };
+
+            string activityFunctionName = nameof(TestActivities.Hello);
+
+            using (JobHost host = TestHelpers.GetJobHost(this.loggerFactory, nameof(Orchestration_Activity)))
+            {
+                await host.StartAsync();
+
+                // Null input should result in ArgumentNullException in the orchestration code.
+                var client = await host.StartFunctionAsync(orchestratorFunctionNames[0], null, this.output);
```

Comment:
Is this comment correct? I looked through the code briefly and I didn't see any place where `OrchestratorGreeting` or `SayHelloWithActivity` throw an exception.